### ISSUE TITLE
Large numbers of teaser_check_failed 

### ISSUE THREAD

**Issue by plukevdh** [wvanbergen/request-log-analyzer#67]
As of release 1.9.2, I'm seeing a lot more teaser_check_failed warnings in the logs. It also is producing radically different output. This is parsing a Rails 2.3.x log file.

It appears to have started as of commit (from `git bisect` output):

commit 18abcf0d77ac05c1b0797aeb0e3e52c777f3da9c
Author: Willem van Bergen willem@vanbergen.org
Date:   Fri Oct 1 10:56:41 2010 +0200

```
Fixed Rails2 format parser to still match :completed lines when ActiveRecord is disabled.
```

**Comment by wvanbergen** [comment]
Ouch. Do you have a line that now triggers a teaser check failed? This way I can add it to the test suite.

**Comment by plukevdh** [comment]
Here you go: http://gist.github.com/624600

**Comment by plukevdh** [comment]
And a fix

http://github.com/plukevdh/request-log-analyzer/commit/4c137910affd67172f66c8d8e8f94e62de87db5b#diff-0

**Comment by wvanbergen** [comment]
Your fix broke some of the other cases that can occur. I have fixed your fix and released it as version 1.9.4
